### ISSUE TITLE
Antonym Interlinking (kṛṣṇá vs. śveta)

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/GRA#4]
kṛṣṇá in GRA contains `(Gegens. śvetā́n)`. We could try to interlink the antonyms, because śveta is such. But it's śveta and not śvetā́n in the dictionary. Sp we should remove the accent and plural ending. Can we? Should we?

![krsna](https://user-images.githubusercontent.com/80761/71319095-843f4280-24aa-11ea-9ec7-58f381afef00.PNG)

**Comment by drdhaval2785** [comment]
I do not think this requires work. Let us not link antonyms, which requires deciphering German singular / plural suffices.

**Comment by funderburkjim** [comment]
Aside from the difficulty of interpreting German, etc.  There is a more general feature that this request exemplifies:  Interlinking of any sort.

Here is an example where a similar 'interlinking' might be considered.

There are in MW numerous `<ab>id.</ab>` references, and some sort of
interlinking would be useful here.   There would be needed some sort markup indicating
the target.  A glib possible such markup might be `<ab ref="X">id.</ab>` 
or possibly `<ref n="X"><ab>id.</ab></ref>`  where X is probably the SLP1 spelling of another headword.

One might do things in such a way that the Antoynym relinking could be handled in the
same way.  Perhaps `<ref n="Sveta">śvetā́n</ref>`.

In either case, 
* X has to be determined, and 
* display programs need modification to do something useful when such markup is encountered.

**Comment by gasyoun** [comment]
> X has to be determined

Let's start with MW? Links to `<ab>id.</ab>` or Panini are long wanting ones.
